data_processors/pipeline/lambdas/orchestrator.py

```diff
@@ -215,6 +217,13 @@ def next_step(this_workflow: Workflow, skip: dict, context=None):
             logger.info("Performing SOMALIER_EXTRACT_STEP")
             results.append(somalier_extract_step.perform(this_workflow))
 
+        # ToDo
+        if "STAR_ALIGNMENT_STEP" in skiplist:
+            logger.info("Skip performing STAR_ALIGNMENT_STEP")
+        else:
+            logger.info("Performing STAR_ALIGNMENT_STEP")
+            results.append(star_alignment_step.perform(this_workflow))
+
```

Comment:
Don't think we need it here, Flo..? This block says "after DRAGEN WTS analysis ..."

Reply:
Ups. Fixed.